tests/snappi/test_vlan_traffic.py

```diff
@@ -0,0 +1,154 @@
+import pytest
+from pprint import pprint as pprint
+import time
+
+vlan_id = "10"
+macs = ["00:1A:C5:00:00:01", "00:1B:6E:00:00:01"]
+
+
+@pytest.fixture(scope="module", autouse=True)
+def discovery(npu):
+    npu.objects_discovery()
+
+
+def test_l2_untagged_vlan_traffic(npu, dataplane):
+    """
+    Creates vlan 10 and adds two ports to the vlan 10 member
+    and validates the config using l2 traffic
+    """
+    commands = [
```

Comment:
Few things:
1. In case p2 and p3 are untagged members of default vlan 1, they should be removed from vlan 1 first
2. I do not see PVID to be set. Not sure how it may work without it for untagged member

https://github.com/opencomputeproject/SAI-Challenger/blob/53b32cc7ed6035735aa5b59e08ccc758a72909f2/tests/test_l2_basic.py#L20

Reply:
But the current script is passing without any issue., i dont think we need to delete from default vlan1